lib/openmic.rb

```diff
@@ -0,0 +1,50 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+
+  def initialize(data)
+       @location = data[:location]
+       @date = data[:date]
+       @performers = []
+  end
+
+  def welcome(user)
+       @performers << user
+  end
+
+  def repeated_jokes?
+       # require 'pry' ; binding.pry
+
+       joke_ids = []
+       rep_jokes = false
+
+       if @performers.length > 0
+            @performers.each do |performer|
+                 # binding.pry
+                 performer.each do |user|
```

Comment:
This might be where your syntax error is - since a performer is a User object (not an array or hash), you won't be able to call `.each` on it - here is where you should have `performer.jokes.each {|joke| joke_ids << joke.id }`